src/modules/meteor/assets/templates/start.sh

```diff
@@ -29,21 +31,70 @@ docker run \
   <% for(var option in logConfig.opts) { %>--log-opt <%= option %>=<%= logConfig.opts[option] %> <% } %>\
   <% for(var volume in volumes) { %>-v <%= volume %>:<%= volumes[volume] %> <% } %>\
   <% for(var args in docker.args) { %> <%= docker.args[args] %> <% } %>\
+  <% if(typeof sslConfig.autogenerate === "object")  { %> \
+    -e "VIRTUAL_HOST=proclaim.32past8.com" \
```

Comment:
Sorry, that's my host I was using to test with.  I'll update the PR to remove that.